src/components/WizardContainer/index.js

```diff
@@ -0,0 +1,144 @@
+import React, { useEffect, useContext } from 'react';
+import { withRouter } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { Wizard, Steps, Step } from 'react-albus';
+import Step1 from '../Step1';
+import Step2 from '../Step2';
+import WizardContext from '../../context/WizardContext';
+
+import styles from './index.module.scss';
+
+const stepContainers = [
+  {
+    id: 'step1',
+    content: Step1,
+  },
+  {
+    id: 'step2',
+    content: Step2,
+  },
+];
+
+const WizardContainer = (props) => {
+  const { match: { url }, history } = props;
+  const [activeStep, setActiveStep] = useContext(WizardContext);
+
+  const listenToScroll = () => {};
+
+  const push = (step) => {
+    const path = `${url}/${step.id}`;
+    history.push(path);
+  };
+
+  const goTo = (step, params) => {
+    push(step, params);
+    setActiveStep(step);
+  };
+
+  useEffect(() => () => {
+    // every time wizard step changes this code works
+    window.scrollTo(0, 0);
+    // clear wizard error
+  }, [activeStep]);
+
+  useEffect(() => {
+    const redirectHome = () => {
+      history.push('/home');
+      // clear wizard error here
+    };
+
+    const pathToStep = () => {
+      const { pathname } = history.location;
+      const id = pathname.replace(`${url}/`, '');
+      const [step] = stepContainers.filter((s) => s.id === id);
+
+      return step;
+    };
+
+    const step = pathToStep();
+
+    if (!step) {
+      redirectHome();
+    }
+
+    setActiveStep(activeStep);
+
+    window.addEventListener('scroll', listenToScroll);
```

Comment:
Why do we add this listener in effect each time? also it seems to be doing nothing?